crates/ruff/src/checkers/ast/mod.rs

```diff
@@ -2879,30 +2879,30 @@ where
 
                 // flake8-use-pathlib
                 if self.settings.rules.any_enabled(&[
-                    Rule::PathlibAbspath,
-                    Rule::PathlibChmod,
-                    Rule::PathlibMkdir,
-                    Rule::PathlibMakedirs,
-                    Rule::PathlibRename,
+                    Rule::OsPathAbspath,
```

Comment:
The name seems reasonable to me. Should we rename the `StringDotFormat` rule to `StringFormat` for consistency or rename these rules to `OsDotPathDotAbspath` (feels excessive to me)